lib/dgram.js

```diff
@@ -639,6 +639,25 @@ Socket.prototype.unref = function() {
   return this;
 };
 
+
+Socket.prototype.setRecvBufferSize = function(size) {
+  var err = this._handle.setRecvBufferSize(size);
+
+  if (err) {
+    throw errnoException(err, 'setRecvBufferSize');
```

Comment:
@DamienOReilly add the new error code at https://github.com/nodejs/node/blob/master/lib/internal/errors.js#L164 (you can put placeholders in the message like in https://github.com/nodejs/node/blob/master/lib/internal/errors.js#L121